walletSetup.ts

```diff
@@ -43,29 +43,32 @@ import {
   FSCircuitStorage,
   AbstractPrivateKeyStore,
   CredentialStatusPublisherRegistry,
-  Iden3SmtRhsCredentialStatusPublisher
+  Iden3SmtRhsCredentialStatusPublisher,
+  Iden3OnchainSmtCredentialStatusPublisher,
+  OnChainRevocationStorage
 } from '@0xpolygonid/js-sdk';
 import path from 'path';
 import dotenv from 'dotenv';
 dotenv.config();
 import { MongoDataSourceFactory, MerkleTreeMongodDBStorage } from '@0xpolygonid/mongo-storage';
 import { MongoMemoryServer } from 'mongodb-memory-server';
 import { MongoClient, Db } from 'mongodb';
-
-const circuitsFolder = process.env.CIRCUITS_PATH as string;
-const mongoDbConnection = process.env.MONGO_DB_CONNECTION as string;
-
-export function initInMemoryDataStorage({
-  contractAddress,
-  rpcUrl
-}: {
-  contractAddress: string;
-  rpcUrl: string;
-}): IDataStorage {
-  const conf: EthConnectionConfig = defaultEthConnectionConfig;
-  conf.contractAddress = contractAddress;
-  conf.url = rpcUrl;
-
+import { ethers } from 'ethers';
+
+const circuitsFolder = process.env.CIRCUITS_PATH ?? '';
+const mongoDbConnection = process.env.MONGO_DB_CONNECTION ?? '';
+const rpcUrl = process.env.RPC_URL ?? '';
+const chainId = process.env.RHS_CHAIN_ID ? +process.env.RHS_CHAIN_ID : undefined;
```

Comment:
Doesn't seem like this is doing anything...

Reply:
So it got replaced in favour of the config file!